tests/acceptance/bootstrap/SettingsContext.php

```diff
@@ -553,4 +553,56 @@ public function userDisablesAutoAcceptingUsingSettingsApi(string $user): void {
 		$this->featureContext->setResponse($response);
 		$this->featureContext->rememberUserAutoSyncSetting($user, false);
 	}
+
+	/**
+	 * @When user :user disables email notification using the settings API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userDisablesEmailNotificationUsingTheSettingsAPI(string $user): void {
+		$response = HttpRequestHelper::sendRequest(
+			$this->featureContext->getBaseUrl() . "/api/v0/settings/values-save",
+			$this->featureContext->getStepLineRef(),
+			'POST',
+			$this->featureContext->getActualUsername($user),
+			$this->featureContext->getPasswordForUser($user),
+			null,
+			json_encode($this->getBodyForNotificationSetting($user)),
+		);
+		$this->featureContext->setResponse($response);
+	}
+
+	/**
+	 * @param string $user
+	 *
+	 * @return array
+	 */
+	public function getBodyForNotificationSetting(string $user): array {
+		$settingsValues = (json_decode(
+			SettingsHelper::getSettingsList(
+				$this->featureContext->getBaseUrl(),
+				$this->featureContext->getActualUsername($user),
+				$this->featureContext->getPasswordForUser($user),
+				$this->featureContext->getStepLineRef(),
+			)->getBody()->getContents()
+		));
+		foreach ($settingsValues->bundles[0]->settings as $settingsValue) {
+			if ($settingsValue->name === "disable-email-notifications") {
+				return [
+					"value" => [
+						"accountUuid" => "me",
```

Comment:
could you check which is the correct one?
https://github.com/owncloud/ocis/blob/b48b113d7e84e4b4025c3660bb89f920d29579cd/tests/acceptance/bootstrap/SettingsContext.php#L499